spec/system/employee_system_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Employee management', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  # rubocop:disable RSpec/ExampleLength
+
+  it 'enables me to sign up' do
+    visit root_path
+    click_link 'Register'
+    fill_in 'employee[email]', with: 'test@test.pl'
+    fill_in 'employee[password]', with: 'zaq123'
+    fill_in 'Password confirmation', with: 'zaq123'
+    click_button 'Sign up'
+
+    expect(page).to have_text('Logged as: test@test.pl')
+  end
+
+  it 'enables me to sign in' do
+    FactoryBot.create :employee,  email: 'test5@test.pl',
+                                  password: 'zaq123',
+                                  password_confirmation: 'zaq123'
```

Comment:
You have created an employee's factory but you do not use it and you enter the login details yourself :)

It was helpful for me: https://devhints.io/factory_bot

Reply:
Exactly - try building an employee with the factory, and use it's parameters for registration and login